lib/model/query/analytics.js

```diff
@@ -17,7 +17,7 @@ const _cutoffDate = sql`current_date - cast(${DAY_RANGE} as int)`;
 // Gets a pointer to the project (repeat) in the metrics report for a specific
 // project, creating it first from the project metrics template if necessary
 const _getProject = (projects, id) => {
-  if (!Object.hasOwnProperty.call(projects, id)) {
+  if (!projects[id]) {
```

Comment:
we make all our checks explicit in -backend (and i think frontend too), so this needs `!= null` rather than just the negation. js has a lot of bad behavior around implied falsiness.